collector/drbd_linux.go

```diff
@@ -184,7 +187,7 @@ func (c *drbdCollector) Update(ch chan<- prometheus.Metric) error {
 	file, err := os.Open(statsFile)
 	if err != nil {
 		if os.IsNotExist(err) {
-			log.Debugf("drbd: %s does not exist, skipping: %s", statsFile, err)
+			level.Debug(c.logger).Log("msg", "drbd stats file does not exist, skipping", "file", statsFile, "err", err)
```

Comment:
We can drop the `drbd` prefixing now that we have the `collector` tag in the log line. :-)

```suggestion
			level.Debug(c.logger).Log("msg", "stats file does not exist, skipping", "file", statsFile, "err", err)
```